### ISSUE TITLE
Slider does not always goes to the full start/end and instead select text

### ISSUE THREAD

**Issue by vvo** [sneas/img-comparison-slider#23]
Hey there, thanks for your library, super useful for me (I'll send you the website once I get it published).

When using the slider, there's a small quirk I'd like your advice with, here it is in the video:

https://user-images.githubusercontent.com/123822/116557401-b19dcb00-a8fe-11eb-8811-90df79913e1f.mp4

In this video, I start by grabbing the handle with a click and never release the click. But we can still see that if I move fast then the slider does not go to the end of the image. Even when moving slowly it does not really go at the very beginning/end of the image sides.

And instead, sometimes it will select text around.

Is there a way to prevent this you think?

Thanks!

**Comment by sneas** [comment]
Hi Vincent, thank you for describing the issue in detail and making the video. I really appreciate that! I don't know how to help you to fix the issue right away, thus I'm going to do some research in the upcoming days. I'll keep you updated.

**Comment by sneas** [comment]
Thanks for the issue, mate. I'm looking forward to seeing it action on the website you're working on.

**Comment by vvo** [comment]
Hey there thanks a lot for the release, I actually went custom after understanding that what I needed was a slider that would not display a before/after but rather display either one or the other image.

Here's what I needed to build:

https://user-images.githubusercontent.com/123822/117305075-b4f50180-ae7e-11eb-8cc1-f0ef660cca24.mp4

And I realized I could not build it with your library later on.. So I used https://github.com/desandro/draggabilly and did a lot of HTML/CSS on my side.

Thanks again!
